src/coreclr/src/pal/src/locale/utf8.cpp

```diff
@@ -2158,7 +2158,7 @@ class UTF8Encoding
 
         // assume that JIT will enregister pSrc, pTarget and ch
 
-        for (;;) {
+        while (true) {
```

Comment:
I had few for -> while changes in a local branch, which was done for consistency/readability and to avoid a C++ formatter to format `for (;;)` to `for (; ;)`. I started this branch from there. The compiler was not complaining about it.